boards/px4/fmu-v5x/init/rc.board_sensors

```diff
@@ -49,11 +49,11 @@ then
 	fi
 fi
 
-if ver hwtypecmp V5X000000 V5X000001 V5X000002 V5X001000 V5X004000 V5X004001 V5X004002
+if ver hwtypecmp V5X000000 V5X000001 V5X000002 V5X001000 V5X004000 V5X004001 V5X004002 V5X005001 V5X005002
```

Comment:
What about Rev 3 sensors (V5X005003) ?

Reply:
I only added the combination holybro shipped with. For 5x. We only shipped rev 1 and rev2.

Reply:
> What about Rev 3 sensors (V5X005003) ? 

I don't want to add other rev because I'm not sure how much memory it will take.